listenbrainz/webserver/views/views_utils.py

```diff
@@ -0,0 +1,36 @@
+from flask import current_app
+from flask_login import current_user
+
+from listenbrainz.domain.spotify import SpotifyService
+from listenbrainz.domain.youtube import YoutubeService
+
+
+def get_current_spotify_user():
```

Comment:
I wonder if we can turn these into flask extensions, does it make sense? If so, open a ticket and link it to LB-879